diff --git a/node_modules/offline-github-changelog/lib/addCommitSummary.js b/node_modules/offline-github-changelog/lib/addCommitSummary.js
index 8f7fa41..8c24679 100644
--- a/node_modules/offline-github-changelog/lib/addCommitSummary.js
+++ b/node_modules/offline-github-changelog/lib/addCommitSummary.js
@@ -1,6 +1,6 @@
 const { extend } = require('@transformation/core');
 
-const MAX_REGULAR_COMMITS_PER_RELEASE = 5;
+const MAX_REGULAR_COMMITS_PER_RELEASE = Number.POSITIVE_INFINITY;
 
 const addCommitSummary = () =>
   extend({
diff --git a/node_modules/offline-github-changelog/lib/repositoryUrl.js b/node_modules/offline-github-changelog/lib/repositoryUrl.js
index 72db76e..a3bb350 100644
--- a/node_modules/offline-github-changelog/lib/repositoryUrl.js
+++ b/node_modules/offline-github-changelog/lib/repositoryUrl.js
@@ -8,6 +8,7 @@ const repositoryUrl = ({ originName }) =>
     concat(),
     (url) => url.trim(),
     (url) => url.replace(/^git@github.com:/, 'https://github.com/'),
+    (url) => url.replace(/^ssh:\/\/git@github.com:/, 'https://github.com/'),
     (url) => url.replace(/.git$/, '')
   );
 
